package.json

```diff
@@ -9,24 +9,24 @@
     "url": "https://github.com/apiaryio/protagonist.git"
   },
   "engines": {
-    "node": ">=4"
+    "node": ">=12"
   },
   "dependencies": {
-    "nan": "^2.13.2"
+    "nan": "^2.17.0"
   },
   "devDependencies": {
     "async": "^2.6.1",
-    "chai": "^4.1.2",
+    "chai": "^4.3.7",
     "glob": "^7.1.2",
-    "mocha": "^5.2.0",
-    "node-gyp": "^4.0.0"
+    "mocha": "^10.2.0",
+    "node-gyp": "^9.3.0"
   },
   "scripts": {
     "build": "node-gyp build",
     "prepack": "cp drafter/THIRD_PARTY_LICENSES.txt THIRD_PARTY_LICENSES.txt",
     "test:exclude-await": "mocha --exclude test/await-test.js",
     "test:await": "mocha",
-    "test": "node --version | grep -q -E \"v(4|6)\" && npm run test:exclude-await || npm run test:await"
+    "test": "npm run test:await"
```

Comment:
When async/await support had been added to Protagonist it was only usable in NodeJS >6 (since Node <=6 doesn't support async/await). As we've dropped support, this expression to not run async-await tests are no longer needed.